docs/index.md

```diff
@@ -1,17 +1,17 @@
-# Welcome to the NewPipe Documentation.
+# Welcome to NewPipe.
 
 <img width=150 src="https://raw.githubusercontent.com/TeamNewPipe/NewPipe/dev/assets/new_pipe_icon_5.png"/>
 
 
-This site is/should be a beginner friendly tutorial and documentation for people who want to use, or write services for the [NewPipe Extractor](https://github.com/TeamNewPipe/NewPipeExtractor) however it does also contain several notes about how to maintain NewPipe.
+This site is/should be a beginner friendly tutorial and documentation for people who want to use or write services for the [NewPipe Extractor](https://github.com/TeamNewPipe/NewPipeExtractor). However, it also contains several notes about how to maintain NewPipe.
 It is an addition to our auto generated [jdoc documentation](https://teamnewpipe.github.io/NewPipeExtractor/javadoc/).
 
-Please be aware that it is also in an early state, so help and [feedback](https://github.com/TeamNewPipe/documentation/issues) is always welcome :D
+Please be aware that it is in its early stages, so help and [feedback](https://github.com/TeamNewPipe/documentation/issues) is always appreciated :D
 
 
 ## Introduction
 
-The NewPipeExtractor is a Java framework for scraping video platform websites in a way that they can be accessed like a normal API. The extractor is the core of the popular YouTube and streaming App [NewPipe](https://newpipe.schabi.org) for android, however it's system independent and also available for other platforms as well. 
+The NewPipeExtractor is a Java framework for scraping video platform websites in a way that they can be accessed like a normal API. The extractor is the core of the popular YouTube and streaming app [NewPipe](https://newpipe.schabi.org) for Android. It is entirely independent from said platforms and also available for additional platforms as well. 
 
-The beauty behind this framework is it takes care of the extracting process, error handling etc., so you can take care about what is important: Scraping the website.
+The beauty behind this framework is that it takes care of the extracting process, error handling etc. so you can focus on what is important: Enjoying the content.
```

Comment:
Why did you use _enjoying the content_ here? IMHO this does not match the original meaning of _scraping the content_.

Reply:
Na it must keep Scraping. Enjoying does not fit here. This sentense is not ment from the view of the user, but from the view of the developer. The developer should focus on what is important about what he does, and for him it is important to scrape the content.